tests/wpt/mozilla/tests/mozilla/bluetooth/requestDevice/filter-matches.html

```diff
@@ -54,7 +54,6 @@
         test_promises = test_promises
             .then(() => window.navigator.bluetooth.requestDevice(args))
             .then(device => {
-                //assert_in_array(matching_services[0], device.uuids);
```

Comment:
We cant call `device.uuids` here, because the `uuids` idl attribute was not implemented for the `BluetoothDevice` interface. Meanwhile this `uuids` attribute get removed form the specification so I removed it from the tests.